app/styles/maps.less

```diff
@@ -0,0 +1,28 @@
+.leaflet-control-zoom {
+  box-shadow: none;
+  a.leaflet-control-zoom-in, a.leaflet-control-zoom-out {
+    margin-bottom: 4px;
+    border-radius: 0;
+    background-color: @background-main-color;
+    color: #fff;
+    font-size: 20px;
+    font-weight: lighter;
+  }
+}
+
+.leaflet-popup {
+  .leaflet-popup-close-button {
+    display: none;
+  }
+  .leaflet-popup-content-wrapper {
+    border: 2px solid @border-main-color;
+    color: @base-color;
+  }
+  .leaflet-popup-content {
+    font-size: 14px;
+    margin: 6px 20px;
+  }
+  .leaflet-popup-tip {
+    border: 2px solid @border-main-color;
```

Comment:
I would add a background-color: @base-color so there is not a line between the content and the tip.